app/src/main/java/com/arctouch/codechallenge/util/Extensions.kt

```diff
@@ -0,0 +1,28 @@
+package com.arctouch.codechallenge.util
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import com.arctouch.codechallenge.util.Constants.Companion.API_KEY
+import com.arctouch.codechallenge.util.Constants.Companion.BACKDROP_URL
+import com.arctouch.codechallenge.util.Constants.Companion.BUNDLE_ARGS
+import com.arctouch.codechallenge.util.Constants.Companion.POSTER_URL
+
+fun buildPosterUrl(posterPath: String): String {
+    return POSTER_URL + posterPath + "?api_key=" + API_KEY
```

Comment:
Here you can use the template concatenation of Kotlin, like that:

    return "$POSTER_URL$posterPath?$api_key=$API_KEY"

Reply:
Great! 👍